client/post-editor/post-editor.jsx

```diff
@@ -307,7 +324,10 @@ const PostEditor = React.createClass( {
 								maxWidth={ 1462 } />
 							{ this.renderNotice() }
 							<div className="editor__header">
-								<EditorTitleContainer
+								<EditorTitle
+									isNew={ this.state.isNew }
+									post={ this.state.post }
+									site={ site }
```

Comment:
With the new connect approach in EditorTitle it might be nice to get rid of passing the site object in here and perhaps just call [isPermalinkEditable](https://github.com/Automattic/wp-calypso/blob/248979e89810b70d027d166300e8897f57638692/client/post-editor/editor-title/index.jsx#L97) here instead and pass that resultant boolean as a prop in its lieu.